### ISSUE TITLE
`value` inserted before imported variables

### ISSUE THREAD

**Issue by beark** [hosseinmd/prettier-plugin-jsdoc#163]
Input:
```ts
import { something } from './index';
```

Output:
```ts
import { value something } from './index';
```

Expected:
```ts
import { something } from './index';
```

I don't think this plugin should affect imports at all, not quite sure what's going on here.

Tested on 0.3.36 with following config:
```json
{
  "jsdocDescriptionWithDot": true,
  "jsdocSingleLineComment": false,
  "jsdocSeparateTagGroups": true,
  "jsdocPreferCodeFences": true,
  "tsdoc": true
}
```
Works as expected if I roll back to 0.3.31, haven't tried any of the versions in-between.

**Comment by hosseinmd** [comment]
Could you give me your file whole code

**Comment by beark** [comment]
The example input was my entire file. Or could the output depend on more things than just the configuration and the file being formatted (eg, the contents of the file being imported above)?

**Comment by hosseinmd** [comment]
I added a test, but that was work correctly
https://github.com/hosseinmd/prettier-plugin-jsdoc/commit/cb91a5cae01927f2e935aea82b78f4ec09412300

Could you reproduce it in codesandbox

**Comment by Dan503** [comment]
I've had this issue as well.

I haven't been able to replicate it in the tests either. I can also confirm that rolling back to a previous version fixes the issue.

It's a very strange bug. I'm wondering if maybe there is a version conflict somewhere. Possibly different versions of prettier are installed or something. I'm not sure what is causing this yet.

**Comment by hosseinmd** [comment]
On cicd tests running with various version of prettier. All tests passed. Do you have another prettier plugin or ESLint plugin?

**Comment by Dan503** [comment]
Once my [`@default` PR](https://github.com/hosseinmd/prettier-plugin-jsdoc/pull/160) goes through I'll investigate this more thoroughly.

This issue is preventing me from updating to the latest version.

**Comment by Dan503** [comment]
Ok the issue was introduced in **v0.3.35**

v0.3.34 - Did **not** have this issue
v0.3.35 - **Did** have this issue

**Comment by Dan503** [comment]
**Updating to Prettier v2.6.2 fixes the issue!**

The project that had the issue was running prettier v2.2.1 and had no other prettier plugins installed.

I have installed the latest version of prettier and it seems to have fixed the issue for me.

---
PS. In v0.3.35 `src/index.ts` `jsdocParser` version was updated I think. That might have had something to do with the issue. Not sure.

![image](https://user-images.githubusercontent.com/10610368/162731467-5f4654a2-4534-4916-967f-bbcb3a2dd9f6.png)

**Comment by Dan503** [comment]
@beark if updating to prettier v2.6.2 fixes the issue then this issue can probably be closed.

**Comment by beark** [comment]
That does seem to fix things, yes!

**Comment by wardellbagby** [comment]
Hey, just wanted to pop in and say I ran into this as well and I _think_ I have some more information.

So I was running Prettier as an ESLint plugin as well as running Prettier in WebStorm.

When I ran Prettier via its own CLI or via ESLint, the `value` wouldn't be added to the import. However, running Prettier via the IDE _would_ show this issue.

An IDE restart fixed it for me.

This [StackOverflow issue](https://stackoverflow.com/questions/70564950/vscode-prettier-adds-value-to-imports-in-typescript-react) says this might also occur in VS Code and also recommends just restarting your IDE.

In short, this might've never been related to this plugin at all.
